Modules/SubModule_Import/xASL_imp_NII2BIDS_Subject.m

```diff
@@ -32,25 +32,23 @@
     
     % Make a subject directory
     subjectDirectory = fullfile(imPar.BidsRoot,['sub-' bidsLabel.subject]);
-    if xASL_exist(subjectDirectory,'dir')
-        fprintf('Subject %s exists in the rawdata BIDS directory already...\n', ['sub-' bidsLabel.subject]);
-    else
+    if ~xASL_exist(subjectDirectory,'dir')
         xASL_adm_CreateDir(subjectDirectory);
     end
     
     
     %% 2. Process the anat & perfusion files
     % Subsequent code is based on having data per ASL scan, so we "fool" it
-    % by renaming all sessions into ASL_1 ASL_2 ASL_n and keeping the
-    % unique sessions only. Missing scans will issue a warning, not an error.
-    listSessions = xASL_adm_GetFileList(fullfile(imPar.TempRoot,nameSubjectSession),'^(ASL|T1w|FLAIR).+$',false,[],true);
-    listSessions = cellfun(@(y) y(end), listSessions, 'UniformOutput', false);
-    listSessions = unique(listSessions);
-    listSessions = cellfun(@(y) ['ASL_' y], listSessions, 'UniformOutput', false);
+    % by renaming all runss into ASL_1 ASL_2 ASL_n and keeping the
+    % unique runss only. Missing scans will issue a warning, not an error.
```

Comment:
`runss` -> remove one s

Reply:
same above, did you do a find and replace? :)

Reply:
6848c013